apps/plotly_examples/src/components/DeclarativeChart.tsx

```diff
@@ -149,19 +149,21 @@ export class DeclarativeChartBasicExample extends React.Component<{}, IDeclarati
         >
           Download
         </button>
-        <br />
-        <br />
-        <h2>{this.state.selectedChoice}. {selectedSchema.layout.title}</h2>
-        <br />
-        <br />
-        <ErrorBoundary>
-          <DeclarativeChart
-            key={uniqueKey}
-            chartSchema={inputSchema}
-            onSchemaChange={this._handleChartSchemaChanged}
-            componentRef={this._declarativeChartRef}
-          />
-        </ErrorBoundary>
+        <div data-testid="chart-container" >
```

Comment:
Can you regenerate all charts. I added error boundary. 
The failing ones should also generate a screenshot now.

Reply:
im working on it Atisay. i will update once it is done.